pages/clusters/overview.md

```diff
@@ -27,57 +24,43 @@ Clusters encapsulate groups of agents and pipelines, enabling the following:
+- Type of work: open source vs everything else
+- Type of platform/build: Linux, Android, macOS, Windows, Docker, ML, etc
+- Product lines: companies with multiple products often have a cluster configured for each individual product.
 
 You can create as many clusters as your require for your setup.
 
 ### How should I structure my queues
 
-Queues should mimic your infrastructure. The most common queue attributes we see customers use are:
+The most common queue attributes are based on infrastructure set-ups, such as:
 
-- Architecture (x86, arm64 etc)
+- Architecture (x86, arm64, Apple silicon, etc.)
 - Size of agents (small, medium, large)
-- Type of machine (Mac, GPU, Linux, Windows)
+- Type of machine (macOS, Linux, Windows, GPU, etc.)
 
-So an example queue would be `small_mac_silicon`.
+Therefore, an example queue would be `small_mac_silicon`.
 
-Having individual queues according to these breakdowns allows you to scale your agents that all look the same and Buildkite will report on these
+Having individual queues according to these breakdowns allows you to scale your agents that all look the same and Buildkite will report on these.
 
 ## Queue metrics
 
 Clusters provides additional, easy to access queue metrics that are available only for queues within a cluster. Learn more in [Cluster queue metrics](/docs/pipelines/cluster-queue-metrics).
 
 ## Accessing clusters and agents
 
-The release of clusters changes how your agents are accessed through the Buildkite interface.
-
 If you only have one Cluster with one Queue selecting _Agents_ in the global navigation will take you to your single queue.
 
-If you have multiple clusters, or unclustered pipelines and agents, selecting _Agents_ in the global navigation will take you to the _Agent Clusters_ page. Once on this page, you can navigate to your agents by selecting the cluster the agents are part of, or _Unclustered_ for agents that were not created as part of a cluster.
-
-## Enabling clusters before the release date
-
-Any Buildkite administrator can enable clusters for an organization. Once you enable clusters, you can only disable them by contacting support.
-
-Enabling clusters also changes access to agent tokens. Rather than being available in the Buildkite dashboard, agent tokens are only visible upon creation, ensuring greater security for your applications.
-
-To enable clusters:
-
-1. Securely save any existing agent tokens you need because these won't be available after enabling clusters.
-1. Navigate to your [organization’s pipeline settings](https://buildkite.com/organizations/~/pipeline-settings).
-1. In _Clusters_, select _Enable Clusters_.
+If you have multiple clusters, or unclustered pipelines and agents, selecting _Agents_ in the global navigation will take you to the _Clusters_ page. Once on this page, you can navigate to your agents by selecting the cluster the agents are part of, or _Unclustered_ for agents that were not created as part of a cluster.
```

Comment:
I'm considering removing this because it's now effectively covered in the following section/page heading...

Reply:
Done